changelogs/unreleased/gh-5860-implement-streams-in-iproto.md

```diff
@@ -0,0 +1,24 @@
+## feature/core
+
+* Streams and interactive transactions over streams are implemented in iproto.
```

Comment:
The text structure is intimidating even when I just look at it. Please, make it less 'monolithic', more structured.

Reply:
Fixed